README.md

```diff
@@ -52,7 +52,56 @@
  - Merge only after a pull request review
  - Deploying immediately after review
  
- #### Tools
+ ### Tools
  
   - `.dotaliases` for git: https://github.com/algotech/dotaliases/blob/master/doc/bash/git_aliases.md
   - SourceTree: https://www.sourcetreeapp.com/
+
+## Code review
+
+A guide for reviewing code and having your code reviewed. We have to accept that many programming decisions are opinions. Discuss tradeoffs, and reach a resolution quickly.
+
+
+### When having your code reviewed
+
+ - Be grateful for the reviewer's suggestions.
+ - Don't take it personally. The review is of the code, not you!
+ - Assume the best intention from the reviewer's comments.
+ - Push commits based on earlier rounds of feedback as isolated commits to the branch. Do not squash until the branch is ready to merge. Reviewers should be able to read individual updates based on their earlier feedback.
+ - Seek to understand the reviewer's perspective.
+ - Try to respond to every comment.
+
+### When reviewing code
+
+ - Understand why the change is necessary (you should at least read the ticket).
+ - Identify ways to simplify the code while still solving the problem.
+ - Seek to understand the author's perspective.
+ - Ask good questions; don't make demands.
+ - Offer alternative implementations, but assume the author already considered them.
+
+### Code review checklist
+
+This code review checklist helps the code reviewers and software developers (during self code review) to gain expertise in the code review process.
+
+#### Architectural
+
+ - Is all the code easily understandable?
+ - Is there any duplicate code?
+ - Is the code as modular as possible?
+ - Are all the queries in repositories?
```

Comment:
Isn't this tip a little bit too specific to a particular type of applications?

Reply:
Completely agree with you here. This is quite specific. Unfortunately, I don't know how to handle this. Should we have 2 different checklists (at the bottom) for frontend and backend? Would that be enough?

Reply:
I am not sure it is a good idea to go that deep into details. Even if you split between front and back, there are so many languages and frameworks each with their own specifics.

Reply:
Yeah ... I think you are right again. Then what about something a bit more generic: `Is all the code in the right place (queries in repositories, data manipulators in managers, etc)?`

Reply:
👍